.github/workflows/linters.yml

```diff
@@ -6,11 +6,11 @@ on:
 
 jobs:
   PyLint:
-    runs-on: ubuntu-20.04
+    runs-on: ubuntu-22.04
```

Comment:
since you're using docker why not `ubuntu-latest` ?

Reply:
Good catch, I changed it to `latest` 👍